ngraph/python/src/ngraph/opset6/ops.py

```diff
@@ -79,3 +79,30 @@ def gather_elements(
     }
 
     return _get_node_factory_opset6().create("GatherElements", inputs, attributes)
+
+
+@nameable_op
+def mvn(
+    data: Node,
+    axes: Node,
+    normalize_variance: bool,
+    eps: float,
+    eps_mode: str,
+    name: Optional[str] = None,
+) -> Node:
+    """Return a node which performs MeanVarianceNormalization (MVN)
```

Comment:
Please, fix the style check issue:

`src/ngraph/opset6/ops.py:93:1: D400 First line should end with a period`